src/style/colors.js

```diff
@@ -123,11 +123,43 @@ class Colors {
       tints.push(tint);
     });
 
-    const sliced = tints.slice(0, 8);
-    return sliced;
+    let sliced = tints.slice(0, 8);
+    const adjusted = adjustSaturation(sliced, color);
+    return adjusted || sliced;
   });
 }
 
+function adjustSaturation(colors, color) {
```

Comment:
Let's add some test to cover these logic

Reply:
Ok